site/soc/software/charts/osh/openstack-tempest/tempest.yaml

```diff
@@ -0,0 +1,100 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: openstack-tempest-soc
+  layeringDefinition:
+    abstract: false
+    layer: site
+    parentSelector:
+      name: tempest-global
+      component: tempest
+    actions:
+      - method: merge
+        path: .
+  storagePolicy: cleartext
+data:
+  wait:
+    timeout: 3000
+{% if deploy_tempest is defined and deploy_tempest is sameas true %}
+  values:
+    # Adding images overrides here instead of versions.yaml
+    images:
+      tags:
+        tempest_run_tests: "{{ suse_osh_registry_location }}/openstackhelm/tempest:{{ suse_infra_image_version }}"
+        ks_user: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    conf:
+      script: "tempest run --config-file /etc/tempest/tempest.conf -w 4 {{ test_args }}"
```

Comment:
Could we have a variable for the number of workers? so we could override it if needed.

Reply:
I created a var in defaults called tempest_workers and set the default value to 4. It can be overridden in extravars if needed.